src/hooks/useCreatePost.js

```diff
@@ -1,20 +1,20 @@
-import React from 'react'
-import axios from 'axios'
+import React from 'react';
+import axios from 'axios';
 
 export default function useCreatePost() {
   const [state, setState] = React.useReducer((_, action) => action, {
     isIdle: true,
-  })
+  });
 
   const mutate = React.useCallback(async (values) => {
-    setState({ isLoading: true })
+    setState({ isLoading: true });
     try {
-      const data = axios.post('/api/posts', values).then((res) => res.data)
-      setState({ isSuccess: true, data })
+      const data = await axios.post('/api/posts', values);
+      setState({ isSuccess: true, data });
```

Comment:
This was a legit bug in that it wasn't waiting for the return. I am tempted to add it back in to see if people notice during the workshop... ;P